x-pack/plugins/cases/public/common/translations.ts

```diff
@@ -257,13 +257,22 @@ export const LINK_APPROPRIATE_LICENSE = i18n.translate('xpack.cases.common.appro
 
 export const CASE_SUCCESS_TOAST = (title: string) =>
   i18n.translate('xpack.cases.actions.caseSuccessToast', {
+    values: { title },
+    defaultMessage: 'A case has been updated: "{title}"',
```

Comment:
@nastasha-solomon this is the "generic" message we'll show as a toast when the user selects or creates a case.

Reply:
@academo this looks good and is generic enough for multiple situations involving case updates. The only recommendation I'd make is to move the `{title}` attribute to the start of the message. This would cut down on the number of words in the message (e.g., `{title} has been updated`) and immediately tells the user which case was updated.

Also, remind me again what non-alert attachments are? Are they things like host IPs and user names?